### PR TITLE
initial review commit all action tested on our setup

### PR DESCRIPTION
**Pull request by jitesh-spryiq**
All action tested on our setup

### CODE REVIEW

File: fresh-service-desk-msp/info.json [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+                       "reply_cc_emails": [],
+                       "spam": false,
+                       "email_config_id": null,
+                       "fr_escalated": false,
+                       "group_id": null,
+                       "priority": 1,
+                       "requester_id": 1232528431,
+                       "requested_for_id": 1000000670,
+                       "responder_id": null,
+                       "source": 2,
```
Comment (cs-shivaji-kolape) [inline fresh-service-desk-msp/info.json]: Output schema should be key with empty values. remove values from response.

File: fresh-service-desk-msp/info.json [hunk 2]
```diff
@@ -0,0 +1,562 @@
...
+                       "status": 2,
+                       "subject": "Support Needed...",
+                       "description": "<div>Details about the issue...</div>",
```
Comment (cs-shivaji-kolape) [inline fresh-service-desk-msp/info.json]: Output schema should consist of keys with empty values. This is applicable to all actions that include values in their responses

File: fresh-service-desk-msp/operations.py [hunk 3]
```diff
@@ -0,0 +1,241 @@
+""" Copyright start
```
Comment (cs-shivaji-kolape) [inline fresh-service-desk-msp/operations.py]: Please use the following copyright content for community submission
"""
Copyright start
MIT License
Copyright (c) 2023 Fortinet Inc
Copyright end
"""

File: fresh-service-desk-msp/info.json (hunk 2 above)
Comment (jitesh-spryiq) [reply fresh-service-desk-msp/info.json]: updaet info.json

File: fresh-service-desk-msp/operations.py (hunk 3 above)
Comment (jitesh-spryiq) [reply fresh-service-desk-msp/operations.py]: update copyright

**Review by cs-shivaji-kolape — APPROVED** [review]
LGTM 👍

**Pull request merged**